src/inference/tests/functional/ngraph_reshape_tests.cpp

```diff
@@ -1,1282 +0,0 @@
-#include <ngraph/op/relu.hpp>
-#include <ngraph/op/result.hpp>
-#include <ngraph/opsets/opset.hpp>
-#include <sstream>
-#include <string>
-#include <vector>
-
-#include "common_test_utils/common_utils.hpp"
-#include "common_test_utils/data_utils.hpp"
-#include "common_test_utils/file_utils.hpp"
-#include "common_test_utils/test_common.hpp"
-#include "ie_common.h"
-#include "openvino/core/partial_shape.hpp"
-#include "openvino/core/shape.hpp"
-
-using namespace testing;
-using namespace InferenceEngine;
-
-using NGraphReshapeTests = ov::test::TestsCommon;
-
-TEST_F(NGraphReshapeTests, getBatchSize) {
-    std::shared_ptr<ngraph::Function> ngraph;
-    {
-        ngraph::PartialShape shape({1, 3, 22, 22});
-        ngraph::element::Type type(ngraph::element::Type_t::f32);
-        auto param = std::make_shared<ngraph::op::Parameter>(type, shape);
-        auto relu = std::make_shared<ngraph::op::Relu>(param);
-        auto result = std::make_shared<ngraph::op::Result>(relu);
-
-        ngraph::ParameterVector params = {param};
-        ngraph::ResultVector results = {result};
-
-        ngraph = std::make_shared<ngraph::Function>(results, params);
-    }
-
-    CNNNetwork cnnNetwork(ngraph);
-    ASSERT_EQ(1, cnnNetwork.getBatchSize());
-}
-
-TEST_F(NGraphReshapeTests, ReshapedDynamicShapeLayout) {
-    std::shared_ptr<ngraph::Function> ngraph;
-    {
-        ngraph::PartialShape shape({-1, 3, 22, 22});
-        ngraph::element::Type type(ngraph::element::Type_t::f32);
-        auto param = std::make_shared<ngraph::op::Parameter>(type, shape);
-        param->set_friendly_name("A");
-        auto relu = std::make_shared<ngraph::op::Relu>(param);
-
-        ngraph::ParameterVector params = {param};
-
-        ngraph = std::make_shared<ngraph::Function>(relu, params);
-    }
-
-    CNNNetwork cnnNetwork(ngraph);
-    ASSERT_EQ(Layout::NCHW, cnnNetwork.getInputsInfo()["A"]->getLayout());
-    ASSERT_EQ(cnnNetwork.getInputsInfo()["A"]->getInputData()->getDims(), (SizeVector{0, 3, 22, 22}));
-
-    ICNNNetwork::InputShapes new_shape;
-    new_shape["A"] = {1, 3, 22, 22};
-    cnnNetwork.reshape(new_shape);
-
-    ASSERT_EQ(Layout::NCHW, cnnNetwork.getInputsInfo()["A"]->getLayout());
-    ASSERT_EQ(cnnNetwork.getInputsInfo()["A"]->getInputData()->getDims(), (SizeVector{1, 3, 22, 22}));
-}
-
-TEST_F(NGraphReshapeTests, CNNReshapeSpatialReLU) {
-    std::shared_ptr<const ngraph::Function> ngraph;
-    {
-        ngraph::PartialShape shape({1, 3, 22, 22});
-        ngraph::element::Type type(ngraph::element::Type_t::f32);
-        auto param = std::make_shared<ngraph::op::Parameter>(type, shape);
-        param->set_friendly_name("data");
-        auto relu = std::make_shared<ngraph::op::Relu>(param);
-        auto result = std::make_shared<ngraph::op::Result>(relu);
-
-        ngraph::ParameterVector params = {param};
-        ngraph::ResultVector results = {result};
-
-        ngraph = std::make_shared<const ngraph::Function>(results, params);
-    }
```

Comment:
@ilya-lavrenov as I can see, we don't have other tests with reshape. Should we port/rework it?

Reply:
I suppose we have https://github.com/openvinotoolkit/openvino/blob/master/src/core/tests/model.cpp#L587-L958

Reply:
@ilya-lavrenov ok, thank you!